lldb/tools/lldb-dap/Protocol/ProtocolRequests.h

```diff
@@ -183,7 +194,7 @@ struct Configuration {
 
   /// Specify a source path to remap "./" to allow full paths to be used when
   /// setting breakpoints in binaries that have relative source paths.
-  std::optional<std::string> sourcePath;
+  std::string sourcePath = "";
```

Comment:
Why not just? Same below.
```suggestion
  std::string sourcePath;
```

Reply:
Removed the `= ""` part here and below.